pkg/reconciler/reconcile.go

```diff
@@ -9,6 +9,36 @@ import (
 	"go.uber.org/zap"
 )
 
+type FleetReconciler struct {
+	// Data required for reconcile phases
+	// For AWS (concurrent safe) & IBM Clients decide whether they are created
+	// and destroyed per each reconcile or globals can be used
+}
+
+func (r *FleetReconciler) initReconciler(ctx context.Context) {
+	// Initializes the fields of FleetReconciler struct
+}
```

Comment:
No need to add artificial code structures before we observe a use for them.

Reply:
done.